meadServer/routes/index.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Now that you have your routes figured out, you can remove this. Alternatively, you can create a routes file like you have here and put your routes into this file. But in that case I'd put your actual functionality in a different file. Semantically, it's nice to have something like

```
app.put('/db', handleNewTemp);
```
and in another file
```
handleNew = async function (req, res){ ...}
```

If you had a lot of routes, you'd end up with a routes file that just specified all the applicable paths in your web app and what functionality gets called in a relatively human-readable format. I.e. it's essentially the interface declaration of your app

The other way to do this is to break up your routes into files based on their domain/purpose like how you have an index route and a users route. I'm not sure which one is more prototypically 'express-y', but the thing I proposed above is more 'rails-y'. (Personally, I think breaking up routes across files is confusing)